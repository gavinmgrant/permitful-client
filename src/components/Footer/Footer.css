footer {
    color: #18181E;
    background-color: #DEC79B;
    margin: 0;
    padding: 15px;
    width: 100%;
    display: flex;
    justify-content: space-between;
}

.contact {
    font-size: 24px;
    margin-left: 5px;
    margin-top: 5px;
    margin-bottom: auto;
}

.contact-icon a {
    padding: 0 10px;
    color: #18181E !important;
    font-size: 1em;
}

.contact-icon:hover {
    opacity: 0.7;
}

.copyright {
    margin-right: 5px;
    margin-top: auto;
    margin-bottom: auto;
    font-size: 16px;
}

@media screen and (min-width: 1024px) {
    .copyright {
        margin-right: 20px;
        font-size: 18px;
    }

    .contact {
        font-size: 32px;
        margin-left: 10px;
    }
}