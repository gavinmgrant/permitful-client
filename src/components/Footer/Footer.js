import React from 'react';
import { Email, Github } from '../../utils/Icons';
import './Footer.css';

export default function Footer() {
    return (
        <footer>
            <section className='contact'>
                <span className='contact-icon'>
                    <a href='mailto: gavingrant@gmail.com' target='_blank' rel='noopener noreferrer'>
                        {Email}
                    </a>
                </span>
                <span className='contact-icon'>
                    <a href='https://github.com/gavinmgrant/permitful-client' target='_blank' rel='noopener noreferrer'>
                        {Github}
                    </a>
                </span>
            </section>
            <section className='copyright'>
                Copyright Â© 2020 <a href='https://gavingrant.dev/' target='_blank' rel='noopener noreferrer' style={{color: '#000000'}}>Gavin Grant</a>
            </section>
        </footer>
    )
}