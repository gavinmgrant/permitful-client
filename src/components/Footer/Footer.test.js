import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import Footer from './Footer';

describe(`Footer component`, () => {
    it('renders a footer by default', () => {
      const wrapper = shallow(<Footer />)
      expect(toJson(wrapper)).toMatchSnapshot()
    })
});