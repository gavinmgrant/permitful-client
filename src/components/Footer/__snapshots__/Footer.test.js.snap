// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component renders a footer by default 1`] = `
<footer>
  Copyright Â© 2020 Permitful
</footer>
`;
